docs/headless-cms/docs/06-01-docstopic.md

```diff
@@ -0,0 +1,87 @@
+---
+title: DocsTopic
+type: Custom Resource
+---
+
+The `docstopics.cms.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to define an orchestrator that creates Asset CRs for a specific asset type. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd docstopics.cms.kyma-project.io -o yaml
+```
+
+## Sample custom resource
+
+This is a sample DocsTopic custom resource (CR) that provides details of the Asset CRs for the **redis**, **asyncapi**, **markdown**, and **openapi** source types.
+
+```
+apiVersion: cms.kyma-project.io/v1alpha1
+kind: DocsTopic
+metadata:
+  name: service-catalog
+  namespace: default
```

Comment:
default is default, no need to mention it

Reply:
Which another meaningful example I could give?